modules/layers/src/scatterplot-layer/scatterplot-layer-vertex.glsl.ts

```diff
@@ -18,7 +18,46 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-export default `\
+import {ShaderUniformType, glsl} from '@luma.gl/core';
+
+export type ScatterplotLayerUniforms = {
+  // opacity: number;
+  radiusScale: number;
+  radiusMinPixels: number;
+  radiusMaxPixels: number;
+  lineWidthScale: number;
+  lineWidthMinPixels: number;
+  lineWidthMaxPixels: number;
+  stroked: number;
+  filled: boolean;
+  antialiasing: boolean;
+  billboard: boolean;
+  radiusUnits: number;
+  lineWidthUnits: number;
+};
+
+export const scatterplot: {uniformTypes: Record<string, ShaderUniformType>} = {
```

Comment:
It's a little confusing to see this constant referenced in the layer code. Maybe name it `ScatterplotUniformLayout`?